src/game.py

```diff
@@ -8,15 +8,15 @@ class Game:
     def __init__(self, ui=UserInterface()):
         self.board = Board()
         self.rules = Rules()
-        self.user_interface = ui
+        self.ui = ui
         self.game_board = self.board.starter_board
         self.player_one = self.board.player_one
         self.player_two = self.board.player_two
         self.total_marks_on_board = 0
         self.play_again = False
 
     def get_formatted_board(self):
-        message.display_formatted_board(self, self.game_board)
+        self.ui.display_message(message.formatted_board(self, self.game_board))
```

Comment:
I would treat displaying a _message_ different from displaying a _board_.  It may feel like duplication, but its not.  Why is that?